yw_website/yw_website/apps/website/models.py

```diff
@@ -102,3 +102,65 @@ class TagFile(models.Model):
 
     class Meta:
         unique_together = ('tag', 'file')
+
+class ProgramBlock(models.Model):
+    class Meta:
+        unique_together = ('id', 'version')
```

Comment:
can you make these `version_id` ?

Reply:
Not really the Django way

Reply:
why not? most databases i've seen by convention name foreign keys with the `id` appended to the end of the table name

Reply:
because it's a model object not sql, it's implied by using a ForeignKey model field

Reply:
otherwise you will end up with an object called version_id who has a id field, and to get that ID you would need to write code like "version_id.id"

Reply:
ok sounds good to me